### PR TITLE
restores the paper assets working on webroot assets

### PR DESCRIPTION
**Pull request by harryob**
we don't use browse_rsc stuff in 2023 please

:cl:
fix: paper images should appear once again
/:cl:

### CODE REVIEW

**Review by Drulikar — APPROVED** [review]
I cannot properly test these changes locally (they worked before w/o these changes - only broken for CDN). But they still work locally w/o a CDN and the code looks good to me.

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -251,9 +253,9 @@
 		t = replacetext(t, "\[/grid\]", "</td></tr></table>")
 		t = replacetext(t, "\[row\]", "</td><tr>")
 		t = replacetext(t, "\[cell\]", "<td>")
-		t = replacetext(t, "\[logo\]", "<img src = wylogo.png>")
-		t = replacetext(t, "\[wy\]", "<img src = wylogo.png>")
-		t = replacetext(t, "\[uscm\]", "<img src = uscmlogo.png>")
+		t = replacetext(t, "\[logo\]", "<img src = [asset.get_url_mappings()["wylogo.png"]]>")
+		t = replacetext(t, "\[wy\]", "<img src = [asset.get_url_mappings()["wylogo.png"]]>")
+		t = replacetext(t, "\[uscm\]", "<img src = [asset.get_url_mappings()["uscmlogo.png"]]")
```
Comment (BeagleGaming1) [inline code/modules/paperwork/paper.dm:258]: You seem to have removed a `>` from this one, was that intended?

Comment (harryob) [inline code/modules/paperwork/paper.dm:258]:
```suggestion
		t = replacetext(t, "\[uscm\]", "<img src = [asset.get_url_mappings()["uscmlogo.png"]]>")
```

**Review by BeagleGaming1 — APPROVED** [review]
code looks good

**Pull request merged**